scripts/release_ppa.sh

```diff
@@ -96,7 +103,7 @@ commitdate=$(git show --format=%ci HEAD | head -n 1 | cut - -b1-10 | sed -e 's/-
 echo "$commithash" > commit_hash.txt
 if [ $branch = develop ]
 then
-    debversion="$version-develop-$commitdate-$commithash"
+    debversion="$version~develop-$commitdate-$commithash"
```

Comment:
Why `~`?

Reply:
It was suggested by a person on launchpad: https://answers.launchpad.net/launchpad/+question/677998